src/views/splash.html

```diff
@@ -0,0 +1,134 @@
+                                                <p>
+                                                    Join us.
+                                                </p>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id="intro-social" class="one unit h">
+                                <h4>
+                                    Social
+                                </h4>
+                                <div>
+                                    <a href="https://www.facebook.com/hackcambridge">
+                                        <img id="splash-facebook-logo" src="{{ asset('images/hc-2018/facebook-logo.svg') }}" alt="Facebook logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://twitter.com/@hack_cambridge">
+                                        <img id="splash-twitter-logo" src="{{ asset('images/hc-2018/twitter-logo.svg') }}" alt="Twitter logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://medium.com/@HackCambridge">
+                                        <img id="splash-medium-logo" src="{{ asset('images/hc-2018/medium-logo.svg') }}" alt="Medium logo" draggable="false"/>
+                                    </a>
+                                    <a href="https://www.instagram.com/hack_cambridge/">
+                                        <img id="splash-instagram-logo" src="{{ asset('images/hc-2018/instagram-logo.svg') }}" alt="Instagram logo" draggable="false"/>
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section id="splash-signup" class="red">
+                <div class="container width">
+                    <div class="one half unit row">
+                        <div class="six unit h"></div>
+                        <div class="seven unit h">
+                            <div class="container">
+                                <div class="one unit row">
+                                    <div class="five unit h">
+                                        <h5>
+                                            Sign up now!<br><span class="secondary">We’ll be in touch soon.</span>
+                                        </h5>
+                                    </div>
+                                </div>
+                                <div class="half unit row">
+                                    <div class="five unit h">
+                                        <form id="splash-signup-form" class="half unit row" method="post" action="some-action">
```

Comment:
The styling of this form looks off — for example, the distance between the placeholder text and the line underneath it seems too small, and the submit button seems small.  If this is something you'd prefer to work on in a future pull, that makes sense but please create an issue to track it.